reference/Upload-API.yaml

```diff
@@ -17,11 +17,11 @@ tags:
 servers:
   - url: 'https://cloudmatrix.upload.streamamg.com'
 paths:
-  /upload:
+  /entry:
     parameters: []
-    put:
+    post:
       summary: Request an Upload URL
```

Comment:
should be, create a new CloudMatrix entry

Reply:
Done